gusto/time_discretisation/wrappers.py

```diff
@@ -381,9 +387,13 @@ def setup(self):
         # -------------------------------------------------------------------- #
         # Set up test function
         # -------------------------------------------------------------------- #
+        if self.time_discretisation.wrapper_field_names is not None:
+            uadv = Function(domain.spaces('HDiv'))
```

Comment:
could this line be above the if statement?

Reply:
Done, put it below